Makefile

```diff
@@ -11,6 +11,16 @@ all: \
 	_output/bin/linux/bootkube \
 	_output/bin/darwin/bootkube \
 	_output/bin/linux/checkpoint \
+	_output/bin/linux/amd64/checkpoint \
+	_output/bin/linux/arm/checkpoint \
+	_output/bin/linux/arm64/checkpoint \
+	_output/bin/linux/ppc64le/checkpoint \
+	_output/bin/linux/s390x/checkpoint
+
+build: \
```

Comment:
Added this because it takes forever for `make all` to cross-compile every flavor and this is handy. We can rearrange these however people like.

Reply:
It would be super awesome if `all` only built for your arch, and `release` built for all arches. But I am terrible at make and so this is fine if that approach is too hard.

Reply:
There's no difficulty to do that. Currently our "release" (bootkube.tar.gz) only includes:

* bin/linux/bootkube
* bin/darwin/bootkube
* bin/linux/checkpoint

The cross compilation is meant to pave the way for creating binaries for docker images for each architecture.

Maybe `all` should remain as-is to stay fast, and the full list of possible binaries should be in a temporary target like `cross`, at least until we have a target for the docker image. Right now the docker builds are done by a pile of scripts.

Reply:
Sure, that's fine. I think in the future if `make release` includes all the arches, then it'd be nicer for `make all` to only include my current arch. Making darwin every time increases my build time by ~33%